py_ecc/fields/field_elements.py

```diff
@@ -49,8 +50,9 @@ def __init__(self: T_FQ, val: IntOrFQ) -> None:
             self.n = val % self.field_modulus
         else:
             raise TypeError(
-                "Expected an int or FQ object, but got object of type {}"
-                .format(type(val))
+                "Expected an int or FQ object, but got object of type {}".format(
```

Comment:
Might be nice to make these f-strings at some point?

Reply:
Got'em, plus some %-strings.